src/organisms/cards/ContactCard/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import icons from 'assets/stylesheets/utilities/_icons_font.scss';
+import styles from './contact_card.module.scss';
+import classnames from 'classnames';
+
+function ContactCard ( props ) {
+  const {
+    phoneNumber,
+    className
+  } = props;
+
+  return (
+    <ul className={ classnames(className, styles.contact_card) }>
+      <li><a className={ classnames(icons['i-phone']) } href="#">{ phoneNumber }</a></li>
+      <li><a className={ classnames(icons['i-chat']) } href="#">Chat with an Expert</a></li>
+    </ul>
+  );
+}
+
+ContactCard.propTypes = {
+  /**
+   * This prop will add a new className to any inherent classNames
+   * provided in the component's index.js file.
+   */
+  className: PropTypes.string,
+  /**
+   * You know what a phone number is.
```

Comment:
maybe the prop isn't specifically a phone number, maybe it's a top and bottom text? that would make this component really flexible